superset/views/base.py

```diff
@@ -139,15 +141,11 @@ def get_error_msg() -> str:
 
 
 def json_error_response(
-    msg: Optional[str] = None,
+    msg: str | None = None,
     status: int = 500,
-    payload: Optional[dict[str, Any]] = None,
-    link: Optional[str] = None,
```

Comment:
`link` parameter is not used in the codebase at all

Reply:
@sebastianliebscher it seems like the `payload` argument isn't use in the codebase either.

Reply:
there is a single usage here https://github.com/apache/superset/blob/d95c200e6931e2bc14b0448e3da265d8ebedf249/superset/views/core.py#L166